packages/cli/src/tools/config/index.js

```diff
@@ -106,7 +106,7 @@ function loadConfig(projectRoot: string = process.cwd()): ConfigT {
           (project, platform) => {
             project[platform] = this.platforms[platform].projectConfig(
               projectRoot,
-              userConfig.project[platform],
+              userConfig.project[platform] || {},
```

Comment:
could you fix that better typings?

Reply:
We already have the good typings. Flow can't handle them. userConfig.project is a map, we just need to handle missing keys.

Reply:
iOS/Android is always defined because we use Joi schema validation and apply some defaults to not need to handle with that.